examples/serve/https/cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDbTCCAlWgAwIBAgIUOPkOX89x3GWlrI9mOsi/6kIZ4kQwDQYJKoZIhvcNAQEL
+BQAwRTELMAkGA1UEBhMCVVMxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoM
+GEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAgFw0yNDA2MDUwMDMxMjNaGA8yMTI0
+MDUxMjAwMzEyM1owRTELMAkGA1UEBhMCVVMxEzARBgNVBAgMClNvbWUtU3RhdGUx
+ITAfBgNVBAoMGEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDCCASIwDQYJKoZIhvcN
+AQEBBQADggEPADCCAQoCggEBAN3hLSyXnU89evgCXxpGJ59/mgbuKZZZvmo2LDRt
+OGyd/BD3JvjaY0KRzblT8PajLkXUkp9dbQIlhspjVIzMvp8yUxNVMOlPqnmIFLkY
+jP4EhaNt9GCBaPnYH7QPCMVU5agxyrZfOj5GKpMbcBZUbnzxNqO+8cAv/hWuIMhw
+w8eh6C4LPJRWwOhDGN+Q19bOgGFgOzMQXVVxP+9OChldNL6SlsqVtUg80fJS1lkJ
+gtzg/c0MyU1C/QAco93X47bNqz9S98xByl4wE2Si5C2nhkj2W+O/QvCXGCFHPWzY
++KkV6Bkh/Z26d+5IdLz0fKj8DPXGssdUmoTPBYwf5LhBDC0CAwEAAaNTMFEwHQYD
+VR0OBBYEFPR9eczYOyaPmGLVS+TDI3UKcj9yMB8GA1UdIwQYMBaAFPR9eczYOyaP
+mGLVS+TDI3UKcj9yMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEB
+ANh2jVJLWvqdDPi9YowVGfUFKpGbeoeyiShIQQmQ69bJKO8sZPEor6kJZwlbU0OQ
+95jHGnoK7caoV9OFYGvkXxTX9im2QfsX9sWl2lcMzGXZEM9qFnJRGDZl6nEcLTQP
+2ijnq0pOjNkfdajcmP2/E2x1y5eHP8QeQzTpDaHMuCpNxAw+849J2uM2LNh4HZQa
+sxiJs7t4qun0ZVqDxdlzbXlj53Xkab3xwoa1s1svRIHiNeVO3eHq2gYkrzvxoK6o
+J9Lw6CdrOzo0bKxC6mB5uNHT9RRzePOpwn5NRgePpLGjq/VdZhtRdROoSjNN7wx9
++bo9qmPp2EcRP6t6BidFWuY=
+-----END CERTIFICATE-----
```

Comment:
It is weird to include private key in a repo. How about we leave out these keys, only having the instruction for key generation in the YAML?

Reply:
That would be needed if we want to include smoke tests..

Reply:
Can't we call the key generation command in smoke test?